### DISCUSSION TITLE
How to validate Radio Button

### DISCUSSION THREAD

**Discussion by finnoplan** [dockwa/simple-react-validator#413] [Q&A]
Hi I have radio button as input. But their no example of radio button validation

**Comment by stuyam** [comment]
The way Simple React Validator works is that it just validates data. So if for example you are just requiring a radio button to have a selected value, and you start storing the value of what gets selected in state then you would just do something like:
```jsx
<input type="radio" name="size" value="small" checked={this.state.size === 'small'} onChange={this.updateState}> Small
<input type="radio" name="size" value="medium" checked={this.state.size === 'medium'} onChange={this.updateState}> Medium
<input type="radio" name="size" value="large" checked={this.state.size === 'large'} onChange={this.updateState}> Large
{this.validator.message('size', this.state.size, 'required')}
```

So when `size` does not have a value (aka it has not been selected by the user) it will show a validation message when you trigger messages to show. Once a value is set for `this.state.size` then it will be valid because it will satisfy the `required` requirement.
